frontend/components/ProjectCard.vue

```diff
@@ -1,12 +1,12 @@
 <template>
-  <div class="project-card" @click="handleClick">
-    <div class="project-name">
-      <span class="icon">🚀</span>
+  <div class="border border-gray-300 rounded-lg p-4 bg-white shadow mb-4 cursor-pointer transition-transform duration-300 ease-in-out hover:shadow-lg hover:scale-105" @click="handleClick">
```

Comment:
I am not familiar with Tailwind CSS, but shouldn't we factor the style items (border, rounded, etc) into a declared CSS class to factor code ?

Reply:
No, actually the philosophy of Tailwind is to be [utility-first](https://tailwindcss.com/docs/utility-first) (as opposite to Atomic or BEM methodologies) to be easily customizable and overridable. If you really want to factorize a specific style you should create reusable components instead (as you will usually also want to factorize the associated HTML structure anyway).